bin/emqx

```diff
@@ -912,6 +1034,11 @@ case "${COMMAND}" in
         ;;
 
     upgrade|downgrade|install|unpack|uninstall)
+        ## For now, we don't have relup; remove once we are ready
+        ## TODO: always deny relup for macos (unsupported)
+        logerr "Unsupported upgrade path"
+        exit 1
```

Comment:
Er, we can't release it like this.
What I meant is to do this in the `main` function in install_upgrade.escript.
Otherwise if we release `bin/emqx` this way for `e5.0.0`, there is no way to upgrade from it.

Reply:
:hear_no_evil: :see_no_evil:

Reply:
:heavy_check_mark: 
https://github.com/emqx/emqx/pull/9612/commits/465e73cac0176fb9af1837c8e93d2c423bc9be89